src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -12,78 +12,84 @@
 import edu.wpi.first.math.kinematics.SwerveDriveOdometry;
 import edu.wpi.first.math.kinematics.SwerveModulePosition;
 import edu.wpi.first.math.kinematics.SwerveModuleState;
-import edu.wpi.first.wpilibj.DriverStation;
 import edu.wpi.first.wpilibj.SPI.Port;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.SwerveConfig;
+import frc.robot.SwerveModule;
+import frc.robot.constants.CompConstants;
 import frc.robot.constants.SwerveConstants;
 
 public class SwerveSubsystem extends SubsystemBase {
   /** Creates a new SwerveSubsystem. */
-  private SwerveDriveKinematics _kinematics;
-  private SwerveDriveOdometry _odometry;
+  private final SwerveDriveKinematics mKinematics;
+  private final SwerveDriveOdometry mOdometry;
 
-  private AHRS _navx;
-
-  private SwerveModule[] _modules;
-  public static double tuningOutput = 0;
-  public static double autoGyroInitValue = 0;
+  private final AHRS mNavX;
 
+  private final SwerveModule[] mModules;
+  public static double mAutoGyroInitValue = 0;
```

Comment:
Should this really be a static? This seems broken to me.

Reply:
Oh and it's public, which makes it even more likely to be broken. Indicates a nasty hack me thinks...

Reply:
I can make a public getter and setter for this and make the field private.

Reply:
And not static?

Reply:
yep, the only instance of this field changing is in a class where the drivebase object is provided, so it does not need to be static.